testutils/application.go

```diff
@@ -0,0 +1,85 @@
+package testutils
```

Comment:
maybe rename to `cliotestutils` for easy import / not clashing with other test utils packages?